yt/yt/tests/integration/proxies/test_rpc_proxy.py

```diff
@@ -853,6 +853,36 @@ def test_sort(self):
 
         assert select_rows("* from [//tmp/t_out] LIMIT " + str(2 * size)) == original_table
 
+    @authors("faucct")
+    def test_writing_large_rows(self):
+        data = [{"index": 0, "str": "F" * (17 << 20)}]
+        create("table", "//tmp/table", attributes={"dynamic": False, "schema": self._schema})
+        write_table("//tmp/table", data, table_writer={"max_row_weight": 20 << 20})
+        assert read_table("//tmp/table") == data
+
+    @authors("faucct")
+    def test_writing_large_rows_129(self):
+        data = [{"index": 0, "str": "F" * (129 << 20)}]
+        create("table", "//tmp/table", attributes={"dynamic": False, "schema": self._schema})
+        with raises_yt_error(f'Expected <= {128 << 20}, found {130 << 20}'):
+            write_table("//tmp/table", data, table_writer={"max_row_weight": 130 << 20})
+        with raises_yt_error() as e:
+            write_table("//tmp/table", data, table_writer={"max_row_weight": 128 << 20})
+        assert 'Memory limit exceeded while parsing YSON stream: ' in str(e[0]) or 'Row weight is too large' in str(e[0])
```

Comment:
were you able to figure out what causes one or the other error to trigger? from the looks of the code we should hit one of them first, and that the memory limit in yson parser is pretty strict

but since both errors are from server code I guess it is fine..

Reply:
No, I wasn't. Most of the time I was getting the second error, but got the first error once and did not want to risk the test flapping.